Formula/soapysdr.rb

```diff
@@ -0,0 +1,34 @@
+class Soapysdr < Formula
+  desc "Vendor and platform neutral SDR support library"
+  homepage "https://github.com/pothosware/SoapySDR/wiki"
+  url "https://github.com/pothosware/SoapySDR/archive/soapy-sdr-0.8.0.tar.gz"
+  sha256 "cff3b1fb1cd3b88932f37df42f69fc3d8e0ea42f4788b35131ceaf5d1b908ac2"
+  license "BSL-1.0"
+  head "https://github.com/pothosware/SoapySDR.git"
+
+  depends_on "cmake" => :build
+  depends_on "swig" => :build
+  depends_on "python@3.9"
+
+  def install
+    args = []
+    args += ["-DENABLE_PYTHON=OFF"]
+    args += ["-DENABLE_PYTHON3=ON"]
+    args += ["-DCMAKE_MODULE_LINKER_FLAGS_INIT='-undefined dynamic_lookup'"]
+    args += %W[-DSOAPY_SDR_ROOT='#{HOMEBREW_PREFIX}']
```

Comment:
```suggestion
    args = std_cmake_args + %W[
      -DENABLE_PYTHON=OFF
      -DENABLE_PYTHON3=ON
      -DCMAKE_MODULE_LINKER_FLAGS_INIT=-undefined\ dynamic_lookup
      -DSOAPY_SDR_ROOT=#{HOMEBREW_PREFIX}
    ]
```

Reply:
As mentioned in my comment, I am thinking some of these args are unnecessary.

At least the CMake linker flags is probably copied from upstream formula https://github.com/pothosware/homebrew-pothos/commit/b0e380b0c158745f96af2d2e5e1a4fa86fdaec8f, which should be related to build fix committed into https://github.com/pothosware/SoapySDR/commit/a8df1c575ff7345fe72050c18ede8c545e255c46

Reply:
Yup, agree. Needing `-undefined dynamic_lookup` is also a bit of a smell.

Reply:
The fix for -undefined dynamic_lookup was done after the 0.8.0 release

Reply:
According to https://github.com/pothosware/SoapySDR/commit/a8df1c575ff7345fe72050c18ede8c545e255c46, the commit is in 0.8.0.

Unless there is another missing commit, the above fix was from Jul 13, 2020 while 0.8.0 was released on Apr 25, 2021.

Reply:
Didn't realize the years were different 😅

Reply:
Can you use the style shown in the above suggestion?
The `]` can be indented at same level with `args`

You can also see if CI style check will fail by running `brew style soapysdr`